velox/functions/lib/TimeUtils.h

```diff
@@ -31,6 +31,12 @@ inline constexpr int64_t kSecondsInDay = 86'400;
 inline constexpr int64_t kDaysInWeek = 7;
 extern const folly::F14FastMap<std::string, int8_t> kDayOfWeekNames;
 
+FOLLY_ALWAYS_INLINE bool isTimeUnit(const DateTimeUnit unit) {
+  return unit == DateTimeUnit::kMillisecond || unit == DateTimeUnit::kSecond ||
+      unit == DateTimeUnit::kMinute || unit == DateTimeUnit::kHour ||
+      unit == DateTimeUnit::kMicrosecond;
```

Comment:
nit: perhaps reorder the `kMicrosecond` before `kMillisecond` to improve readability.